src/app/modules/pictureapp-explore/components/nav/nav.component.html

```diff
@@ -4,8 +4,8 @@
             <mat-icon>menu</mat-icon>
   </button>
   <span>
-      <button mat-button >
-          PictureApp         
+      <button mat-button [routerLink]="['/']" (click)="emitdashboardAsCurrentSection()"> 
```

Comment:
```suggestion
      <button mat-button [routerLink]="['/']" (click)="emitDashboardAsCurrentSection()"> 
```
To be followed with convention ;)

Reply:
ok fixed name convention